src/Parser/Paladin/Retribution/Modules/Features/AlwaysBeCasting.js

```diff
@@ -47,17 +48,38 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
     SPELLS.HAND_OF_RECKONING.id,
   ];
 
+  get suggestionThresholds() {
+    return {
+      actual: this.downtimePercentage,
+      isGreaterThan: {
+        minor: 0.1,
+        average: 0.2,
+        major: 0.3,
+      },
+      style: 'percentage',
+    };
+  }
+
   suggestions(when) {
-    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+    const {
+      actual,
+      isGreaterThan: {
+        minor,
+        average,
+        major,
+      },
+    } = this.suggestionThresholds;
+    const boss = this.owner.boss;
 
```

Comment:
You dont have to add this in anymore, instead just say
``when(this.suggestionThresholds).addSuggestion`` then you can also remove this ``.regular(average).major(major)`` like you do in HolyPowerDetails